files/image_config/procdockerstatsd/procdockerstatsd.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Process and docker CPU/memory utilization data export daemon
+Requires=updategraph.service
+After=updategraph.service
```

Comment:
Since this daemon also needs the Redis database to be up and running, please change these lines as follows`:

```
Requires=database.service updategraph.service
After=database.service updategraph.service
```